ui/src/views/Tasks/TaskGroup/index.jsx

```diff
@@ -591,12 +590,12 @@ export default class TaskGroup extends Component {
 
     // Task groups do not necessarily have a decision task,
     // so handle task-not-found errors gracefully
-    return (
-      error.graphQLErrors &&
-      error.graphQLErrors[0] &&
-      error.graphQLErrors[0].statusCode === 404 &&
-      error.graphQLErrors[0].requestInfo.method === 'task'
-    );
+
+    // return (
+    //   error.graphQLErrors &&
+    //   error.graphQLErrors[0].statusCode === 404 &&
+    //   error.graphQLErrors[0].requestInfo.method === 'task'
+    // );
```

Comment:
Instead of commenting this code out, we should iterate over the array and check if there are any entry errors that fit the conditions (i.e., `statusCode` is `404` and method is 'task'). We can probably use the [`some`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some) to do this.

Reply:
alright will work on that